.github/workflows/ci.yml

```diff
@@ -5,6 +5,20 @@ on:
   pull_request:
 
 jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Fetch the src
+      uses: actions/checkout@v3
+    - name: Set up Python 3.11
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.11
+
+    - name: Install requirements
+      run: python -m pip install -r test_requirements.txt
```

Comment:
Installing the requirements is not enough. The most important part is running the tests — please, add a step for running Django tests through `manage.py`. It will run 0 tests as there's none but exit with a successful return code.

Reply:
Added in https://github.com/kpi-web-guild/django-girls-blog-Martolivna/pull/19/commits/c32a591aa8a9c97adb244fe4adf7087e816c8643 .

Reply:
Marking as resolved.